docker-compose.yml

```diff
@@ -12,8 +12,11 @@ services:
   registry:
     build:
       context: docker/registry
-    pull_policy: build
-    image: sab-cdn-registry:latest
+      tags:
+        - ghcr.io/sab-sachsen/sab-cdn-registry:${TAG:-latest}
```

Comment:
Is that `1.0.1-latest`?

Reply:
Nope, it's the `$TAG` env var, but with a default value if not present `:-`. That's docker magic. https://docs.docker.com/compose/compose-file/compose-file-v3/#variable-substitution